test/test_other.py

```diff
@@ -7296,17 +7296,11 @@ def test_disable_inlining(self):
 }
 ''')
 
-    # Without the 'INLINING_LIMIT', -O2 inlines foo()
-    cmd = [EMCC, '-c', 'test.c', '-O2', '-o', 'test.o', '-sINLINING_LIMIT', '-flto']
+    # Without the 'INLINING_LIMIT', -O2 inlines foo() and then DCEs it because it has
+    # no callers and is static
```

Comment:
Can you verify that this is true but running without `-sINLINING_LIMIT` first?

Reply:
yes, I checked that this test also does what I expected when removing -sINLINING_LIMIT.

Reply:
I meant can the test run the this build twice and verify both the positive and negative case?

Reply:
It could. That would basically amount to a test that the inliner is working at O2 (and that emscripten is passing O2 from emcc into clang). The former should be covered in the LLVM regression tests. Do we have something like the latter in emscripten? Would it be worth it, or is that kind of straight passthrough test not worth it?

Reply:
We are trying to test that `-sINLINING_LIMIT` does something.. so I think its reasonable to very that the output differs from the default.   Basically we want to be sure that that source code and command (sans `-sINLINING_LIMIT`) are such that inlining would occur.  

For example, if I remove the `static` from the code under test, this test would still pass right?   But it would pass for the wrong reason.